app/views/transacts/new.html.erb

```diff
@@ -0,0 +1,10 @@
+<div>
+ <% @header = "NEW TRANSACTION" %>
+  <%= form_with model: @transact, url: category_transacts_path(@category) do |form| %>
+          <div class='main-field'>
+            <%= form.text_field :name,  placeholder: "Name", required: true, class:'text-field'%>
+            <%= form.number_field :amount, placeholder: "Amount (USD)", required: true, class:'text-field' %>
+          </div>
+    <%= form.submit "Save", class:'save-btn'%>
+  <% end %>
+</div>
```

Comment:
- [ ] As stated in the project requirement when creating a new transaction with, indicating their name and amount and categories(Kindly make sure that all are mandatory ).
 - Your new transaction  Form miss the categories as seen in the below screenshot, as stated in the project requirement there should be categories (mandatory at least one)  
 
![image](https://github.com/AlselwiAisha/Budget_App/assets/93598189/a3893b2f-357b-4921-baf8-f74b6a05ed52)

Reply:
I am sorry, I don't get the issue